fibdrv.c

```diff
@@ -122,17 +122,17 @@ static int __init init_fib_dev(void)
     fib_class = class_create(THIS_MODULE, DEV_FIBONACCI_NAME);
 
     if (!fib_class) {
-        printk(KERN_ALERT "Failed to create device class");
+        printk(KERN_ALERT "Failed to create device class\n");
         rc = -3;
         goto failed_class_create;
     }
 
     if (!device_create(fib_class, NULL, fib_dev, NULL, DEV_FIBONACCI_NAME)) {
-        printk(KERN_ALERT "Failed to create device");
+        printk(KERN_ALERT "Failed to create device\n");
         rc = -4;
         goto failed_device_create;
     }
-    return rc;
+    return rc - major;
```

Comment:
Don't change this line since this pull request tends to flush Linux kernel messages immediately.

Reply:
Thanks, I will change the code to cancel this line of code change.